src/benchmarks/gc/GC.Infrastructure/GC.Analysis.API/GC.Analysis.API.csproj

```diff
@@ -2,21 +2,32 @@
 
   <PropertyGroup>
     <ImplicitUsings>enable</ImplicitUsings>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute> 
   </PropertyGroup>
 
+  <Import Project="../Versions.props" />
+
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="2.2.0" />
     <PackageReference Include="Microsoft.Data.Analysis" Version="0.19.1" />
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.7" />
+    <PackageReference Condition="'$(CustomTraceEvent)' != 'true'" Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.9" />
```

Comment:
Was this condition intentional?

Reply:
Yes, it is a feature of this change.

If you look at `Versions.props`, there is a switch where you can flip back and forth using either a Nuget package or a locally build TraceEvent.

This is the enabler for us to quickly iterate different versions of TraceEvent.